scripts/_common.sh

```diff
@@ -161,28 +161,28 @@ ynh_setup_source () {
     fi
     echo "$SOURCE_SUM $SOURCE_FILE" |$SUM_PRG -c --status \
         || ynh_die "Corrupt source"
-    
+
     sudo mkdir -p "$DEST"
-    sudo chown $AS_USER: "$DEST" 
+    sudo chown $AS_USER: "$DEST"
     if [ "$(echo ${SOURCE_FILE##*.})" == "gz" ]; then
         ynh_exec_as "$AS_USER" tar xf $SOURCE_FILE -C "$DEST" --strip-components 1
     elif [ "$(echo ${SOURCE_FILE##*.})" == "bz2" ]; then
         ynh_exec_as "$AS_USER" tar xjf $SOURCE_FILE -C "$DEST" --strip-components 1
     elif [ "$(echo ${SOURCE_FILE##*.})" == "zip" ]; then
         mkdir -p "/tmp/$SOURCE_FILE"
         ynh_exec_as "$AS_USER" unzip -q $SOURCE_FILE -d "/tmp/$SOURCE_FILE"
```

Comment:
I try with zip : but seems there are an issue here : user don't have the right on `/tmp/$SOURCE_FILE` not ls related and not really needed